railties/lib/rails/commands/server.rb

```diff
@@ -67,6 +69,7 @@ def start
       print_boot_information
       trap(:INT) { exit }
       create_tmp_directories
+      create_cache_file if options[:cache]
```

Comment:
It seems like setting `--perform-caching` once never removes the file and leaves caching on all the time. Let's fix that :smile: